scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -0,0 +1,88 @@
+:_content-type: ASSEMBLY
+[id="enabling-workload-partitioning"]
+= Workload partitioning
+include::_attributes/common-attributes.adoc[]
+:context: enabling-workload-partitioning
+
+toc::[]
+
+In resource-constrained environments, you can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+
+The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
+With workload partitioning, you annotate the set of cluster management pods and a set of typical add-on Operators for inclusion in the cluster management workload partition.
+These pods operate normally within the minimum size CPU configuration.
+Additional Operators or workloads outside of the set of minimum cluster management pods require additional CPUs to be added to the workload partition.
+
+Workload partitioning isolates user workloads from platform workloads using standard Kubernetes scheduling capabilities.
+
+The following is an overview of the configurations required for workload partitioning:
```

Comment:
I’m wondering if staring here with the overview, can you put all this into a Procedure module? And list the steps you need to take to configure workload partitioning?

Reply:
Post-review: Shane shared that the updates he made were actually quite small, but because he renamed the file, Git treated it as a totally new file which is how I reviewed it. I told him to feel free to disregard the more robust structural suggestions I made since they're out-of-scope for what he was actually updating.

Reply:
Thanks @skrthomas I've formulated the two steps as more of a procedure and amalgamated some of the content as you suggested below.